src/proxy/index.ts

```diff
@@ -141,6 +144,7 @@ export default class Proxy extends Router {
             content:     workerHammerheadContent,
         });
 
+        this.GET(SERVICE_ROUTES.messaging, (req: http.IncomingMessage, res: http.ServerResponse, serverInfo: ServerInfo) => this._onServiceMessage(req, res, serverInfo));
```

Comment:
The `GET` route for the service messages is necessary only for upgrade WebSocket requests. Let's create a separate handler for this.